apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerCommand.ExecutableCommand.cs

```diff
@@ -436,6 +436,9 @@ private ReadOrQueryRequest GetReadOrQueryRequest()
                     case SpannerCommandType.Read:
                         request = ReadOrQueryRequest.FromReadRequest(GetReadRequest());
                         break;
+                    case SpannerCommandType.Dml: // TODO: This may need to be verified and rectified.
+                        request = ReadOrQueryRequest.FromQueryRequest(GetExecuteSqlRequest());
```

Comment:
I have seen a TODO here and this do appear the closest method that can be used, but we should certainly check it sooner.